### ISSUE TITLE
CUDA out of memory

### ISSUE THREAD

**Issue by ghost** [lzccccc/SMOKE#46]
I have a Nvidia TITAN V with12G of memory. However, when I ran the trainning, it was out of memory:

untimeError: CUDA out of memory. Tried to allocate 240.00 MiB (GPU 0; 11.78 GiB total capacity; 10.69 GiB already allocated; 29.25 MiB free; 19.64 MiB cached)

Is this possible?

**Comment by fzc-mysql** [comment]
I also had the same problem.So I change the batchsize in smoke/configs/smoke_gn_vector.yaml，I can only use "IMS_PER_BATCH: 6" with 1 Nvidia 1080ti GPU.

**Comment by Huleyun** [comment]
> I also had the same problem.So I change the batchsize in smoke/configs/smoke_gn_vector.yaml，I can only use "IMS_PER_BATCH: 6" with 1 Nvidia 1080ti GPU.

Yes, it's ture. my nvidia 2060 with 6G memory can only run under the condition that batch_size=2
